math/genvector/inc/Math/GenVector/Transform3D.h

```diff
@@ -606,36 +868,96 @@ class Transform3D {
    /**
       make transformation from first a rotation then a translation
    */
-   void  AssignFrom( const Rotation3D & r, const Vector & v);
+   void AssignFrom( const Rotation3D & r, const Vector & v)
+   {
+     // assignment  from rotation + translation
+     
+     T rotData[9];
+     r.GetComponents(rotData, rotData +9);
+     // first raw
+     for (int i = 0; i < 3; ++i)
+       fM[i] = rotData[i];
+     // second raw
+     for (int i = 0; i < 3; ++i)
+       fM[kYX+i] = rotData[3+i];
+     // third raw
+     for (int i = 0; i < 3; ++i)
+       fM[kZX+i] = rotData[6+i];
+     
+     // translation data
+     T vecData[3];
+     v.GetCoordinates(vecData, vecData+3);
+     fM[kDX] = vecData[0];
+     fM[kDY] = vecData[1];
+     fM[kDZ] = vecData[2];
+   }
+
 
    /**
       make transformation from only rotations (zero translation)
    */
-   void  AssignFrom( const Rotation3D & r);
+   void  AssignFrom( const Rotation3D & r)
```

Comment:
The idea is that we run clang-format on the PRs first. We have had a .clang-format file describing some of the coding rules but we didn't enforce them so far.

I am worried that if we apply clang-format on all GenVector headers it will go out of the scope of this PR and will make it harder to review.